factcast-store/src/main/java/org/factcast/store/internal/pipeline/TransformingServerPipeline.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright © 2017-2024 factcast.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.factcast.store.internal.pipeline;
+
+import lombok.NonNull;
+import org.factcast.core.Fact;
+import org.factcast.core.subscription.transformation.FactTransformerService;
+import org.factcast.core.subscription.transformation.FactTransformers;
+import org.factcast.core.subscription.transformation.TransformationRequest;
+
+@Deprecated
+// TODO delete
+// This pipeline creates an n+1 query problem on the transformationCache, so that it should not be
+// used
+public class TransformingServerPipeline extends AbstractServerPipeline {
```

Comment:
Class can go away?

Reply:
yes, as soon as the buffered stuffed was proven to be correct. I'll leave it as a fallback for now.